android/libraries/rib-android/src/main/kotlin/com/uber/rib/core/XRay.kt

```diff
@@ -23,70 +23,76 @@ import android.graphics.drawable.BitmapDrawable
 import android.graphics.drawable.Drawable
 import android.view.Gravity
 import android.view.View
+import androidx.annotation.VisibleForTesting
 
 /** Utility class that shows riblets name in its background. */
 class XRay private constructor() {
-  private var isEnabled = false
-  private var textPaint: Paint? = null
```

Comment:
Changed `textPaint` to a lazy `val`.

Reply:
If it's all static info, why not immediately initialize it?

Reply:
`Paint` is a big object. I rather initialize it only if needed. WDYT?

Reply:
BTW, XRay is called in every `ViewRouter` constructor. So, I will initialize `Paint` every time. Even if we don't want to use XRay.

Reply:
@lucasnlm but this is a singleton object, right? So it will really only initialize once. There's only one instance of the `XRay` class, or am I misunderstanding it?

Reply:
"I rather initialize it only if needed. WDYT?"
Ok this makes sense.